pyscriptjs/src/remote_interpreter.ts

```diff
@@ -30,6 +33,13 @@ type PATHInterface = {
     dirname(path: string): string;
 } & ProxyMarked;
 
+type PyScriptPyInterface = ProxyMarked & {
```

Comment:
Also, -0.5 for the name `PyScriptPyInterface`. What about `PyScriptPyModule`?

Moreover, I think this is a broader discussion that we should have, but I am a bit skeptical that this kind of "manual typing" is actually useful: if we remove/change/add functions to `pyscript/__init__.py` they will not be automatically reflected here, so typescript will be happy but we will have a runtime error anyway.
Personally, I think that either static typing works (and gives you strong guarantees that certain kind of bugs cannot be present in your source code) or it's useless.

Reply:
> this kind of "manual typing" is actually useful

This argument is similar to saying that unit tests are useless because the things that you mock can change.

Reply:
Changed the name `PyScriptPyInterface` ==> `PyScriptPyModule`.

Reply:
> we could write our own small wrapper around synclink which just imports/renames/re-exports the types?

+1 for this but not in this PR.